app/src/androidTest/java/ch/hikemate/app/ui/auth/SignInScreenTest.kt

```diff
@@ -0,0 +1,42 @@
+package ch.hikemate.app.ui.auth
+
+import androidx.compose.ui.test.assertHasClickAction
+import androidx.compose.ui.test.assertIsDisplayed
+import androidx.compose.ui.test.assertTextEquals
+import androidx.compose.ui.test.junit4.createAndroidComposeRule
+import androidx.compose.ui.test.onNodeWithTag
+import androidx.test.espresso.intent.Intents
+import ch.hikemate.app.MainActivity
+import com.kaspersky.kaspresso.testcases.api.testcase.TestCase
+import org.junit.After
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+
+class SignInScreenTest : TestCase() {
+  @get:Rule val composeTestRule = createAndroidComposeRule<MainActivity>()
```

Comment:
Question: (Just for my understanding), what is `@get:Rule`?

Reply:
`@get:Rule` annotation indicates that the property composeTestRule is a JUnit test rule